packages/astro-rss/README.md

```diff
@@ -95,6 +95,8 @@ type RSSFeedItem = {
 	pubDate: Date;
 	/** Item description */
 	description?: string;
+	/** Full content of the item, should be valid HTML */
```

Comment:
On behalf of Astro Docs Maintainers, if nothing else changes in the README except this, we're cool!

I'll look forward to seeing a PR to docs in the future. I know a lot of people will be happy to see full post content in RSS, even if only for Markdown files right now.